### PR TITLE
Guess who by Maria So

### PR DESCRIPTION
**Pull request by Bibiruu**
Pull request created.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+          <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
+        </div>
+      </div>  `
+  })
+};                                                                     //board generated in the start function,wont be coming out if called outside
+
+
+const guessWho = () => {                                             //generating the secret person
+  let secret = Math.floor(Math.random() * CHARACTERS.length+1);     
+  selectedCharacter = CHARACTERS[secret];                          //generate random = name in console
```
Comment (AmandaElvkull) [inline script.js:235]: Nice that you explain with comments 👍🏽

File: script.js [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+const checkMyGuess = (personToCheck) => {
+  if (personToCheck === selectedCharacter.name) {                                          
+    winOrLoseText.innerHTML = `You guessed right, it was ${selectedCharacter.name} congratulations!!!!`    
+    console.log(personToCheck)
+  } 
+  else {
+    winOrLoseText.innerHTML = `Sorry better luck next time!! The secret person was ${selectedCharacter.name}`
+    board.style.display = 'none'
+  }
+  winOrLose.style.display = 'flex'                                 // shows wrapper
```
Comment (AmandaElvkull) [inline script.js:415]: same here, nice that with the comments

File: script.js [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+
+// All the DOM selectors stored as short variables
+const tilesSheet = document.querySelector('board');   
+const count = document.getElementById('counter');
+const filter = document.getElementById('filter')            //findout button
+const questions = document.getElementById('questions')
+const winOrLose = document.getElementById('winOrLose')
+const playAgain = document.getElementById('playAgain')
+const board = document.getElementById('board')
+const restartButton = document.getElementById('restart')
```
Comment (AmandaElvkull) [inline script.js:207]: Maybe move them to the top of the file to make it clearer to read the code and good choice of names when creating new DOM selectors.

**Review by AmandaElvkull — COMMENTED** [review]
Overall really good project, meeting the expectation of "blue level". 
Maria has added comments here and there do explain what the function or the outcome should be which is really appreciated when you read the code. 

For next steps for this code can be add counter of guesses ( as i can see that she started with). 

Really good work 👌🏼

File: script.js (hunk 3 above)
Comment (Bibiruu) [reply script.js:207]: Hi Amanda, thank you for the constructive feedback.

**Pull request closed without merging**